tracing-core/Cargo.toml

```diff
@@ -8,7 +8,7 @@ name = "tracing-core"
 #   - README.md
 # - Update CHANGELOG.md.
 # - Create "v0.1.x" git tag.
-version = "0.1.10"
+version = "0.1.9"
```

Comment:
You probably missed this in a rebase but please fix this since `tracing-core 0.1.10` is out: https://docs.rs/tracing-core/0.1.10/tracing_core/